src/tabs/pill.scss

```diff
@@ -5,100 +5,56 @@
 @import './tabs';
 
 @mixin iui-tabs-pill {
-  @include iui-tabs;
-  display: flex;
   flex-flow: row;
   justify-content: space-evenly;
-  margin: $iui-baseline/2 0;
+  border-radius: $iui-border-radius;
```

Comment:
For me it looks odd seeing that small curve on an active tab 😀

Reply:
Fixed it, using `::after { content: ' '; }` instead of `box-shadow` but only on pill tabs.  I could use `::after { content: ' '; }` on all tabs if we'd like to stay consistent.
![Screen Shot 2021-06-16 at 4 06 13 PM](https://user-images.githubusercontent.com/849817/122285485-c76e4c00-cebc-11eb-9fa6-543cec90eb54.png)

Reply:
Check my other comment: https://github.com/iTwin/iTwinUI/pull/156#discussion_r652999113

If we change the style, all these hacks might not be necessary since we can just remove the border-radius

Reply:
`::after` is not really needed here, just remove `border-radius` or set it to 0.